src/main.ts

```diff
@@ -13,18 +13,55 @@ export async function run(): Promise<void> {
     const secrets: string[] = core.getMultilineInput("secrets", {
       required: true,
     });
+    const cloudRegion = core.getInput("cloud_region");
     const baseUrl: string = core.getInput("base_url");
     let identityUrl: string = core.getInput("identity_url");
     let apiUrl: string = core.getInput("api_url");
 
     core.info("Validating bitwarden/sm-action inputs...");
-    if (baseUrl) {
-      if (!isValidUrl(baseUrl)) {
-        throw TypeError("input provided for base_url not in expected format");
+
+    let customUrls;
```

Comment:
It's better to compare against the same type - at the moment it's either `undefined` or `true`:
```suggestion
    let customUrls = false;
```